freqtrade/freqtradebot.py

```diff
@@ -378,7 +378,7 @@ def enter_positions(self) -> int:
             if lock:
                 self.log_once(f"Global pairlock active until "
                               f"{lock.lock_end_time.strftime(constants.DATETIME_PRINT_FORMAT)}. "
-                              "Not creating new trades.", logger.info)
+                              f"Not creating new trades, reason: {lock.reason}.", logger.info)
```

Comment:
while i do understand the reason for this ... do we really need this?

The reason is logged / shown when the lock is added - ain't that enough?

Reply:
I understand the reason is printed when the lock is created, but the lock lasts a long time, so when the logs are checked what you find is that Global pairlock is active, and not the reason. You can go back in the logs and find multiple locks, but it is much more informative if you find the current reason, instead of checking all the locks to find the one that is current.